fatek/tools.py

```diff
@@ -1,6 +1,5 @@
-from .fatek import Fatek
+# from .fatek import Fatek
 
 
 def fatek_cli(adress, symbol):
-
-    print "#TODO console client"
+    print("#TODO console client")
```

Comment:
without `from __future__ import print_function` this won't work in python2. You can also leave `pass` here for now